export/export.go

```diff
@@ -107,28 +107,46 @@ func NewExportService(client clientset.Interface, pods PodService, nodes NodeSer
 	}
 }
 
+type options struct {
```

Comment:
Thanks for taking my suggestion for the functional options pattern. Looks nice. :)